src/GameId.java

```diff
@@ -0,0 +1,3 @@
+public enum GameId {
+    TETRIS, MEMORY, HEARTS
```

Comment:
in accordance to comment below, you need to add here Integer id, constructor and getter for your id.